### ISSUE TITLE
Mailjet::Contact_getcontactslists result is missing crucial informations

### ISSUE THREAD

**Issue by MathieuDerelle** [mailjet/mailjet-gem#155]
we can only see 1 list_id : 
```ruby
y = Mailjet::Contact_getcontactslists.find(16280581)
# => #<Mailjet::Contact_getcontactslists:0x007f9207646158 @attributes={"persisted"=>true, "is_active"=>true, "is_unsub"=>false, "list_id"=>14804, "subscribed_at"=>""}>
```

compared to curl which gives the number of lists subscribed, and the details for each one
```
curl -s -X GET \
--user "X:X" \
https://api.mailjet.com/v3/REST/contact/16280581/getcontactslists

{ "Count" : 6, "Data" : [{ "IsActive" : true, "IsUnsub" : false, "ListID" : 14804, "SubscribedAt" : "" }, { "IsActive" : true, "IsUnsub" : false, "ListID" : 20959, "SubscribedAt" : "" }, { "IsActive" : true, "IsUnsub" : false, "ListID" : 21294, "SubscribedAt" : "" }, { "IsActive" : true, "IsUnsub" : false, "ListID" : 25669, "SubscribedAt" : "" }, { "IsActive" : true, "IsUnsub" : false, "ListID" : 25670, "SubscribedAt" : "" }, { "IsActive" : true, "IsUnsub" : false, "ListID" : 32616, "SubscribedAt" : "" }], "Total" : 6 }%   
```

**Comment by sedubois** [comment]
+1

**Comment by xab3r** [comment]
fixed #194
